VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -88,6 +88,7 @@ export abstract class WaitingRoomBaseDirective {
     displayStartPrivateConsultationModal: boolean;
     displayJoinPrivateConsultationModal: boolean;
     conferenceStartedBy: string;
+    hostWantsToJoinHearing = false;
```

Comment:
Move to host component.

Reply:
If you implement the handleNotConnected() method I mentioned in another comment.